enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -1,73 +1,88 @@
-                        </h2>
-                    </div>
-                    <span class="text">
-                        % if not course_specific:
-                            ${_("Before your account can be linked, you must agree to share your course data with {provider}.").format(provider=enterprise_customer_name)}
-                        % else:
-                            ${_("Before you can access {course_name}, you must agree to share your course data with {provider}.").format(provider=enterprise_customer_name, course_name=course_name)}
-                        % endif
-                    </span>
-                    <div class="form-field data-sharing-consent">
-                        <input
-                            id="register-data_sharing_consent"
-                            type="checkbox"
-                            name="data_sharing_consent"
-                            class="input-block checkbox"
-                            data-consent="${data_sharing_consent}"
-                            data-warning-message="${ messages['warning'] }"
-                        />
-                        <label for="register-data_sharing_consent">
-                            % if not course_specific:
-                                ${_("I agree to allow {platform_name} to share data about my enrollment, completion and performance in all {platform_name} courses and programs where my enrollment is sponsored by {provider}.*").format(provider=enterprise_customer_name, platform_name=platform_name)}
-                            % else:
-                                ${_("I agree to allow {platform_name} to share data about my enrollment, completion and performance in {course_name} with {provider}.*").format(provider=enterprise_customer_name, course_name=course_name, platform_name=platform_name)}
-                            % endif
-                        </label>
-                    </div>
-                    <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }" />
-                    % if course_specific and course_id:
-                        <input type="hidden" name="course_id" value="${ course_id }" />
-                    % endif
-                    % if redirect_url:
-                        <input type="hidden" name="redirect_url" value="${redirect_url}" />
-                    % endif
-                    % if enrollment_deferred:
-                        <input type="hidden" name="enrollment_deferred" value="true" />
-                    % endif
-                    <button type="submit" class="action-primary action">Submit</button>
-                    <p class="note">*${ messages['note'] }
-                </form>
-            </div>
-        </section>
-    </div>
-</main>
+  <%static:css group='style-vendor'/>
+    <%! main_css = "style-main-v2" %>
+<%static:css group='${self.attr.main_css}'/>
+  <%!
+    from django.utils.translation import ugettext as _
+    %>
+<html>
```

Comment:
@haikuginger we need to add a lang attribute to this element. If this page is localized, the two-char lang code should match the users preferred language. It not, it should be lang="en"